packages/devtools_app/lib/src/screens/performance/panes/timeline_events/perfetto/_perfetto_web.dart

```diff
@@ -261,14 +261,14 @@ class _PerfettoViewController extends DisposableController
     _postMessage(message);
   }
 
-  void _handleMessage(html.Event e) {
-    if (e is html.MessageEvent) {
-      if (e.data == EmbeddedPerfettoEvent.pong.event &&
+  void _handleMessage(Event e) {
+    if (e is MessageEvent) {
+      if (e.data.toString() == EmbeddedPerfettoEvent.pong.event &&
```

Comment:
Hah! This is funny, because I think this will work since `toString` is equivalent to `toDart` for JS strings on all backends. Still, we should avoid relying on that and use `toDart` conversions e.g. `(e.data! as JSString).toDart`.